dev-python/time-machine/time-machine-2.9.0.ebuild

```diff
@@ -25,12 +25,5 @@ KEYWORDS="~amd64 ~ppc ~riscv"
 RDEPEND="
 	dev-python/python-dateutil[${PYTHON_USEDEP}]
 "
-BDEPEND="
```

Comment:
I get that it doesn't really change anything but perhaps remove 3.8 from PYTHON_COMPAT to make it more correct.